src/backend/parser/parse_expr.c

```diff
@@ -931,7 +931,22 @@ rewrite_scope_identity_call(ParseState *pstate, Node **lexpr, Node **rexpr)
 	if (!(*lexpr) || !(*rexpr))
 		return;
 
-	if ((IsA(*lexpr, Var) && IsA(*rexpr, FuncExpr)))
+	if (IsA(*lexpr, Var) &&
```

Comment:
Why don't you combine this if clause together with another else clause, I think we really not care about what is the lexpr in here is ?

Reply:
I'm wondering if it is appropriate to write this escape null and like escape '' code in this rewrite_scope_identity_call function. The function's purpose is not to handle the escape null/"" case. I would suggest to create a new function and put all your escape related code there.

Reply:
I have renamed the function as discussed offline with Zhibai.

Reply:
> Why don't you combine this if clause together with another else clause, I think we really not care about what is the lexpr in here is ?

I have updated this in the PR revision.